docs/components/pagination/index.html

```diff
@@ -0,0 +1,144 @@
+---
+title: Pagination
+also:
+  components/buttons: Buttons
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="first-page">First Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+         <button class="hxBtn">4</button>
+        <button class="hxBtn">5</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    </div>
+    {% code 'html' %}
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    {% endcode %}
+  </section>
+
+  <section>
+    <h2 class="hxSectionTitle" id="second-page">Second Page</h2>
```

Comment:
So this isn't the second page selected; it's the third page. Am I misunderstanding what this shows?

Reply:
good catch -- its supposed to be an example of a selected middle page -- fixed to represent that so order should be first, middle, last